content/guides/indicators-timesketch/infrastructure-setup.md

```diff
@@ -8,6 +8,59 @@ cascade:
 
 ## Before we start
 
+Yeti offers two installation methods: Kubernetes or Docker. Choose your
+preferred method and follow the instructions in this guide.
+
+## K8s Installation
+
+If you are new to Kubernetes, consider
+reviewing the OSDFIR Infrastructure [getting started guide](https://github.com/google/osdfir-infrastructure/blob/main/docs/getting-started.md).
+
+To get started, ensure you have
+[Helm](https://helm.sh/docs/intro/install/) and
+[Kubectl](https://kubernetes.io/docs/tasks/tools/) installed and are
+authenticated to your Kubernetes cluster.
+
+> **Note**: For local installations, use
+[Minikube](https://minikube.sigs.k8s.io/docs/start/) or
+[KIND](https://kind.sigs.k8s.io/docs/user/quick-start/).
+
+Once complete, add the repo containing the Helm charts as follows:
+
+```console
+helm repo add osdfir-charts https://google.github.io/osdfir-infrastructure
+```
+
+If you had already added this repo earlier, run `helm repo update` to retrieve
+the latest versions of the packages. You can then run
+`helm search repo osdfir-charts` to see the available charts.
+
+To install the Yeti and Timesketch chart using a release name of `my-release`:
```

Comment:
What's `my-release` here? Is it a local release name, or should it match the release name of the remote chart?